### ISSUE TITLE
Base Idle Clap tolerances too tight

### ISSUE THREAD

**Issue by brainwavecoder9** [TheRobotStudio/SO-ARM100#12]
Probably overkill to add these as issues, and I would consider this VERY minor, but realizing that might be best way to track.

The tolerance on the base idle clamp (pictured) is too tight. When using the 3rd horn on servo 1, the clamp interferes with the horn.

Wouldn't matter if the clamp side of the horn was intended not to move, and I'm sure the servo can overcome it, but the horn is screwed onto the arm so this will cause friction.

![image](https://github.com/user-attachments/assets/5a100642-7201-451d-9152-229cf36132e4)

**Comment by TheRobotStudio** [comment]
So when you screw the clamp on it grips the extra servo horn? It should be slightly loose, just a 0.1mm clearance. If the clamp is not fully screwed down does that allow some movement? If needed I can add a bit more clearance...

**Comment by brainwavecoder9** [comment]
Yes it grips tightly. I was able to carve it out to make clearance.

It's also possible it has to do with my print and/or my material possibly warping. Someone talked me into PC (do not recommend). ABS and PETG have been very good.

**Comment by brainwavecoder9** [comment]
FYI Confirmed the interference with ABS also
